lib/components/artworks/index.js

```diff
@@ -12,8 +12,6 @@ import Relay from 'react-relay';
 import React from 'react-native';
 const { View, ScrollView, StyleSheet } = React;
 
-import { max } from 'lodash';
```

Comment:
While I no longer use lodash, I’m leaving it in because we’re bound to use it again and I already did the work to pin it to the version that some of our dependencies already use.